var/spack/repos/builtin/packages/py-bluepyemodel/package.py

```diff
@@ -12,6 +12,7 @@ class PyBluepyemodel(PythonPackage):
     homepage = "https://bbpgitlab.epfl.ch/cells/bluepyemodel"
     git      = "git@bbpgitlab.epfl.ch:cells/bluepyemodel.git"
 
+    version('0.0.8.2021-12-22', tag='BluePyEModel-v0.0.8')
```

Comment:
Why did we need this? Is the existence of `def patch(self):` not enough to change the hash and force a rebuild?